v22.2/alter-database.md

```diff
@@ -1,34 +1,1082 @@
+~~~
+
+~~~
+CREATE DATABASE
+~~~
+
+#### Set the replica placement policy to `RESTRICTED`
+
+When you set the database's placement policy to [`RESTRICTED`](#parameters-restricted), you are saying that you want the underlying data to be restricted to the table or partition's [home region](multiregion-overview.html#table-localities).
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr PLACEMENT RESTRICTED;
+~~~
+
+~~~
+ALTER DATABASE PLACEMENT
+~~~
+
+#### Set the replica placement policy to `DEFAULT`
+
+If previously you set the replica placement policy to [`RESTRICTED`](#set-the-replica-placement-policy-to-restricted), you can set it back to [the default](#parameters-default) by issuing the following statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE movr PLACEMENT DEFAULT;
+~~~
+
+~~~
+ALTER DATABASE PLACEMENT
+~~~
+
+### Rename database
+
+#### Rename a database
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> CREATE DATABASE db1;
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SHOW DATABASES;
+~~~
+
+~~~
+  database_name
+-----------------
+  db1
+  defaultdb
+  movr
+  postgres
+  system
+(5 rows)
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> ALTER DATABASE db1 RENAME TO db2;
+~~~
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+> SHOW DATABASES;
+~~~
+
+~~~
+  database_name
+-----------------
+  db2
+  defaultdb
+  movr
+  postgres
+  system
+(5 rows)
+~~~
+
+### Configure survival goals
+
+#### Survive zone failures
+
+To change the survival goal of a multi-region database to survive zone failures, use the following statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE {db} SURVIVE ZONE FAILURE;
+~~~
+
+~~~
+ALTER DATABASE SURVIVE
+~~~
+
+{{site.data.alerts.callout_info}}
+Surviving zone failures is the default setting for multi-region databases.
+{{site.data.alerts.end}}
+
+For more information about the zone survival goal, see [Surviving zone failures](multiregion-overview.html#surviving-zone-failures).
+
+#### Survive region failures
+
+To change the survival goal of a multi-region database to survive region failures, use the following statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER DATABASE {db} SURVIVE REGION FAILURE;
+~~~
+
+~~~
+ALTER DATABASE SURVIVE
+~~~
+
+If you try to change a database with less than 3 [database regions](multiregion-overview.html#database-regions) to survive region failures, the following error will be signalled:
+
+~~~
+ERROR: at least 3 regions are required for surviving a region failure
+SQLSTATE: 42602
+HINT: you must add additional regions to the database using ALTER DATABASE mr ADD REGION <region_name>
+~~~
+
+For more information about the region survival goal, see [Surviving region failures](multiregion-overview.html#surviving-region-failures).
+
+## See also
+
+tk
```

Comment:
tk?

Reply:
Hmm, this placeholder should have been updated and I don't see it anymore. So will mark as resolved!